app/templates/welcome.js

```diff
@@ -59,11 +61,17 @@ module.exports = function(state, emit) {
 
   async function upload(event) {
     event.preventDefault();
+    const MAXFILESIZE = config.max_file_size;
```

Comment:
not quite. The `server` path in the require is a clue that we shouldn't use that file here. `MAXFILESIZE` gets set globally in the browser window by https://send.firefox.com/jsconfig.js so we can use it directly here.

Reply:
Using the variable directly gives the following error 
```
67:21  error    'MAXFILESIZE' is not defined  no-undef
  68:7   warning  Unexpected alert              no-alert
  68:64  error    'MAXFILESIZE' is not defined  no-undef
```

And this prevents any commits.
So, we need to do something different I guess

Reply:
This is the linter complaining. Try adding `/* global MAXFILESIZE */` as the first line of the file.

Reply:
Correct, Its working now. I've pushed the commit please have a look.

Reply:
Also, I would squash my commits once the PR is ready to be merged.